Packs/CommonScripts/Scripts/ListUsedDockersImages/ListUsedDockersImages.py

```diff
@@ -0,0 +1,191 @@
+
+    :type max_entries_per_docker: ``int``
+    :param max_entries_per_docker: max number of integration or script to show per docker image entry
+
+    :return: dict as {'docker_image':['integration/scripts', 'integration/scripts',...]}
+    :rtype: dict
+
+    """
+    result = result_dict or {}
+    for integration_script, docker_image in docker_list.items():
+        if integration_script in ['CommonServerUserPowerShell', 'CommonServerUserPython']:
+            continue
+        if docker_image in result:
+            if len(result[docker_image]) < max_entries_per_docker:
+                result[docker_image].append(integration_script)
+        else:
+            result[docker_image] = [integration_script]
+
+    return result
+
+
+def format_result_for_markdown(result_dict: dict) -> list:
+    result_output = []
+    for docker_image, integration_script in result_dict.items():
+        result_output.append({
+            'Docker Image': docker_image,
+            'Integrations/Automations': integration_script
+        })
+    return result_output
+
+
+''' COMMAND FUNCTION '''
+
+
+def get_used_dockers_images(export_to_context: bool = True) -> CommandResults:
+    md = None
+    active_docker_list_integration = {}
+    active_docker_list_automation = {}
+
+    ''' Examples for output: { 'demisto/python3:3.9.7.24076' : ['ListUsedDockerImage', 'VirusTotal',...]}'''
+    result_dict: Dict[str, List[str]] = {}
+
+    active_integration_instances = demisto.internalHttpRequest(POST_COMMAND, '/settings/integration/search',
+                                                               REQUEST_INTEGRATION_SEARCH_BODY)
+    demisto.debug(f'response code = {0}', active_integration_instances['statusCode'])
+    if active_integration_instances and active_integration_instances['statusCode'] == 200:
+        active_docker_list_integration = extract_dockers_from_integration_search_result(
+            active_integration_instances['body'])
+
+    active_automation = demisto.internalHttpRequest(POST_COMMAND, '/automation/search',
+                                                    REQUEST_INTEGRATION_SEARCH_BODY)
+    demisto.debug(f'response code = {0}', active_automation['statusCode'])
+    if active_automation and active_automation['statusCode'] == 200:
+        active_docker_list_automation = extract_dockers_from_automation_search_result(
+            active_automation['body'])
+
+    result_dict = merge_result(active_docker_list_integration, result_dict, MAX_PER_DOCKER)
+    result_dict = merge_result(active_docker_list_automation, result_dict, MAX_PER_DOCKER)
+
+    ''' format the result for Markdown view'''
+    result_output = []
+    result_output = format_result_for_markdown(result_dict)
+
+    md = tableToMarkdown('Dockers Images In use:', result_output)
+
+    if export_to_context:
+        return CommandResults(
+            outputs_prefix='Docker Image',
+            outputs_key_field='Docker Image',
```

Comment:
We don't return fields with space/special chars in them to the context, so it should be DockerImage without spaces.
But it also means you have to either to create similar array but with a bit different keys, or changes the keys to not contain spaces and special chars in them, like `Images`